sequence_decoder.py

```
@@ -319,6 +325,44 @@ def pane_id_from_payload(event_type_name, payload_dict):
+        payload_raw = data[pos:pos + plen]
+        pos += plen
+        if EVENT_TYPES.get(etype) != "PtyOutput":
+            continue
+        try:
+            pd = msgpack.unpackb(payload_raw, raw=False)
+            if isinstance(pd, dict) and len(pd) == 1:
+                inner = next(iter(pd.values()))
+                if isinstance(inner, dict):
+                    pd = inner
+        except Exception:
+            continue
```

Ruff (0.15.21)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)